apps/portlet-clouddrives/src/main/webapp/vue-app/cloudDriveSettings/components/CloudDriveConnector.vue

```diff
@@ -55,6 +56,9 @@ export default {
       });
   },
   methods: {
+    disconnectFromCloudDrive: function(){
+      disconnect(this.userDrive.workspace, this.userDrive.homePath);
```

Comment:
Ensure that this.userDrive already contains the persisted synchronized user cloud drive